pkg/sentry/fsimpl/fuse/read_write.go

```diff
@@ -152,3 +152,92 @@ func (fs *filesystem) ReadCallback(ctx context.Context, fd *regularFileFD, off u
 		fs.conn.mu.Unlock()
 	}
 }
+
+// Write sends a FUSE_WRITE request.
```

Comment:
`Write sends FUSE_WRITE requests and return the bytes written according to the response.`